pkg/cloudprovider/providers/gce/gce_addresses_fakes.go

```diff
@@ -19,12 +19,14 @@ package gce
 import (
 	"fmt"
 	"net/http"
+	"sync"
 
 	compute "google.golang.org/api/compute/v1"
 	"google.golang.org/api/googleapi"
 )
 
 type FakeCloudAddressService struct {
+	sync.RWMutex
```

Comment:
This is not needed for the PR, but I think in general having a lock is still useful as we add more tests.If you rather wait until the lock is needed, I can also remove it.